vhdbuilder/release-notes/autonotes/main.go

```diff
@@ -153,6 +155,16 @@ func getReleaseNotes(sku, path string, fl *flags, errc chan<- error, done chan<-
 		return
 	}
 
+	data, err := os.ReadFile(releaseNotesFileOut)
```

Comment:
it would be nice but realistically that would be a human doing it right now, a bash one liner to remove the "latest" files from the commit should be workable in the short term.